enews-node/app.js

```diff
@@ -0,0 +1,19 @@
+var NewsFetcher = require('./newsFetcher.js')
+
+var token = process.env.token;
+var newsFetcher = new NewsFetcher(token);
+
+var latestDate = new Date();
+var oldestDate = new Date();
+
+oldestDate.setDate(latestDate.getDate() - 31);
+
+// node timestamps are in milliseconds, need to convert to epoch
+var latest = latestDate / 1000;
+var oldest = oldestDate / 1000;
+
+newsFetcher.getEnews(oldest, latest, function(eNews) {
+  eNews.forEach(function(each) {
+    console.log(each);
```

Comment:
each is poor variable name. What is it iterating? chat messages containing the hash tag right, maybe `article` or `eNewsArticle` or `message` etc